flake.nix

```diff
@@ -374,15 +388,42 @@
           '';
         };
 
-        git-hooks-nix = {
-          baseUrl = "https://github.com/cachix/git-hooks.nix/blob/master";
+        pkgs-by-name-for-flake-parts = {
+          title = "pkgs-by-name-for-flake-parts";
+          baseUrl = "https://github.com/drupol/pkgs-by-name-for-flake-parts/blob/main";
           intro = ''
-            Configure pre-commit hooks.
+            [pkgs-by-name-for-flake-parts](https://github.com/drupol/pkgs-by-name-for-flake-parts) is a flake-parts
+            that can autoload Nix packages under a particular directory.
 
-            Generates a configuration for [pre-commit](https://pre-commit.com),
-            provides a script to activate it, and adds a [check](flake-parts.html#opt-perSystem.checks).
+            The structure is identical to the [Nixpkgs one](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md).
 
-            Pre-defined hooks are maintained at [`cachix/git-hooks.nix`](https://github.com/cachix/git-hooks.nix).
+            Quick example how to use it:
+
+            ```nix
+            {
+              inputs = {
+                flake-parts.url = "github:hercules-ci/flake-parts";
+                nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+
+                # (1) add `pkgs-by-name-for-flake-parts` input
+                pkgs-by-name-for-flake-parts.url = "github:drupol/pkgs-by-name-for-flake-parts";
+              };
+
+              outputs = inputs: inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+                systems = [ "x86_64-linux" "aarch64-linux" "aarch64-darwin" ];
+
+                # (2) import pkgs-by-name-for-flake-parts module
+                imports = [
+                  inputs.pkgs-by-name-for-flake-parts.flakeModule
+                ];
+
+                perSystem = { config, self', inputs', pkgs, system, ... }: {
+                  # (3) point to your directory containing Nix packages
+                  pkgsDirectory = ./nix/pkgs-by-name;
+                };
+              };
+            }
+            ```
```

Comment:
I really like that this example is complete.

I'll push a commit to turn off the generated installation instructions (see [preview](https://66ed5dbe2037df4b8182b424--sprightly-cassata-33312f.netlify.app/options/pkgs-by-name-for-flake-parts))

Reply:
[done](https://66ed5f55322d5d4a6a131923--sprightly-cassata-33312f.netlify.app/options/pkgs-by-name-for-flake-parts)